### PR TITLE
Async DR changes

### PR DESCRIPTION
**Pull request by nathan-px**
I'm rolling PD-241 and PD-251 into this PR.

### CODE REVIEW

File: content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md [hunk 1]
```diff
@@ -40,6 +40,65 @@ The following Kubernetes resources are supported as part of the Asynchronous DR
 * ClusterRoleBinding
 * Ingress
 
+## Enable ELB on AWS clusters
+
+If you're running Kubernetes on AWS, you must enable ELB before you can create a cluster pair.
+
+Enable ELB by entering the `kubectl edit service` command and changing the service type value from `nodePort` to `loadBalancer`:
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md]: `LoadBalancer`

There should be a warning here to not enable this with auth enabled on the PX cluster

File: content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md [hunk 2]
```diff
@@ -40,6 +40,69 @@ The following Kubernetes resources are supported as part of the Asynchronous DR
 * ClusterRoleBinding
 * Ingress
 
+## Enable ELB on AWS clusters
+
+If you're running Kubernetes on AWS, you must enable ELB before you can create a cluster pair.
+
+{{<info>}}
+**Warning:** Do not enable ELB with authorization enabled on the PX cluster.
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md]: Sorry, I meant without auth.

File: content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md [hunk 3]
```diff
@@ -40,6 +40,69 @@ The following Kubernetes resources are supported as part of the Asynchronous DR
 * ClusterRoleBinding
 * Ingress
 
+## Enable ELB on AWS clusters
+
+If you're running Kubernetes on AWS, you must enable ELB before you can create a cluster pair.
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md]: This is actually true for any cloud, ie AWS, Google, Azure, IKS

Comment (nathan-px) [reply content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md]: substituted for generic "load balancing" terminology.

**Comment by nathan-px** [comment]
@shailvipx @disrani-px @sensre

File: content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md [hunk 4]
```diff
@@ -40,6 +40,69 @@ The following Kubernetes resources are supported as part of the Asynchronous DR
 * ClusterRoleBinding
 * Ingress
 
+## Enable load balancing on cloud clusters
+
+If you're running Kubernetes on the cloud, you must enable load balancing before you can create a cluster pair.
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md]: `you must configure an External LoadBalancer (ELB) for the PX API service. `

File: content/portworx-install-with-kubernetes/disaster-recovery/shared/cluster-pair.md [hunk 5]
```diff
@@ -2,15 +2,39 @@
...
+  * `--s3-region` with your region
+  * `--s3-endpoint` with `s3.amazonaws.com`
+  * `clusterpair_` with the UUID of your destination cluster
+
+```text
+/opt/pwx/bin/pxctl credentials create --provider s3 --s3-access-key <aws access key> --s3-secret-key <aws secret key> --s3-region us-east-1  --s3-endpoint s3.amazonaws.com clusterpair_<UUID of destination cluster>
+```
+
+{{<info>}}
+**Note:** You can use this command with other object stores, such as Azure and GCP, by specifying a different endpoint and region.
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/disaster-recovery/shared/cluster-pair.md]: The params for the other cloud credentials are different. They don't have endpoint or region.

Comment (nathan-px) [reply content/portworx-install-with-kubernetes/disaster-recovery/shared/cluster-pair.md]: are they entirely different? i.e. do they still start with `--s3-`?

Comment (disrani-px) [reply content/portworx-install-with-kubernetes/disaster-recovery/shared/cluster-pair.md]: They are completely different, they start with `--azure-` and `--google-`

Comment (nathan-px) [reply content/portworx-install-with-kubernetes/disaster-recovery/shared/cluster-pair.md]: so like this?: 

`--provider` as `azure`
  * `--azure-access-key` with your aws access key
  * `--azure-secret-key` with your aws secret key
  * `clusterpair_` with the UUID of your destination cluster

`--provider` as `google`
  * `--google-access-key` with your aws access key
  * `--google-secret-key` with your aws secret key
  * `clusterpair_` with the UUID of your destination cluster

Comment (disrani-px) [reply content/portworx-install-with-kubernetes/disaster-recovery/shared/cluster-pair.md]:
```
      --azure-account-name string      
      --azure-account-key string       
      --google-project-id string       
      --google-json-key-file string    
```

File: content/portworx-install-with-kubernetes/disaster-recovery/shared/cluster-pair.md [hunk 6]
```diff
@@ -2,15 +2,65 @@
...
-#### Generate ClusterPair on the destination cluster
+Perform the following steps to create a cluster pair:
+
+### Create object store credentials for cloud clusters
+
+If you are running Kubernetes on-premises, you may skip this section. If your Kubernetes clusters are on the cloud, you must create object credentials on both the destination and source clusters before you can create a cluster pair.
+
+The options you use to create your object store credentials differ based on which object store you use:
+
+#### Create Amazon s3 credentials
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/disaster-recovery/shared/cluster-pair.md]: @disrani-px I updated these credentials and commands. Are they correct?

Comment (disrani-px) [reply content/portworx-install-with-kubernetes/disaster-recovery/shared/cluster-pair.md]: Yup

**Pull request merged**